modules/developer_manual/examples/core/webdav_api/meta/report-propfind.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<a:propfind xmlns:a="DAV:" xmlns:oc="http://owncloud.org/ns">
+    <a:prop>
+        <a:getcontentlength />
```

Comment:
dont query for props we don't care about

Reply:
So, all of the properties?

Reply:
@DeepDiver1975, which are the essential properties to include?

Reply:
ping @DeepDiver1975

Reply:
@DeepDiver1975, I removed a number of what seem to be unrequired properties. Can you confirm if these are all the ones to be removed, or if others should be removed as well? Thanks.